### DISCUSSION TITLE
is there any way to start Dbeaver with old style SQL highlight?

### DISCUSSION THREAD

**Discussion by Yigitttttttt** [dbeaver/dbeaver#29310] [Q&A] [answered]
is there any way to start Dbeaver with old style SQL highlight?
can i use something on dbeaver.ini or config.ini to do this?

I know i can change it with menu settings.

But I will share it to many users.

it should be on c:/dbeaver folder....

Thanks

**Comment by E1izabeth** [comment, marked as answer]
You can export all settings by the menu File->Export
![image](https://github.com/dbeaver/dbeaver/assets/28875055/35763324-19ad-47af-a55e-457fac0ee862)

Or if you want the concrete one, it's in `DBeaverData\workspace6\.metadata\.plugins\org.eclipse.core.runtime\.settings\org.jkiss.dbeaver.core.prefs` file called `SQLEditor.Highlighting.advanced.enable`
![image](https://github.com/dbeaver/dbeaver/assets/28875055/8aaa4287-5873-4eaf-ae5c-1f0c7c07e512)

**Reply by Yigitttttttt** [reply]
Thanks for reply. 
I tried this ways. I just want to share c:\dbeaver folder and the users shouldn't do anything with settings. So i have to share in c:\dbeaver folder the settings. is it possible? can i change dbeaver.ini or ./configuration/config.ini to have same thing.
(sorry for my English)

**Reply by E1izabeth** [reply]
No, it won't work this way

**Comment by E1izabeth** [comment]
@Yigitttttttt could you also share, why do you want to turn this off?

**Reply by Yigitttttttt** [reply]
For some reasons we still need Toad for Oracle (for F4 partition info lost, Toad is faster sometimes during Excel write for big outputs) and the color change i don't like. Personal preferences actually :)
